api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandler.java

```diff
@@ -12,14 +13,16 @@
 public class ExpiredContractsSqsHandler {
 
     @NonNull private final ContractService contractService;
+    @NonNull private final VoteService voteService;
 
     @SqsListener("ExpiredDonationsSQS-${application.environment}")
     @SuppressWarnings("unused") //Method is invoked when pulling AWS SQS. Method is not directly called within our application.
```

Comment:
Perhaps make this comment a proper javadoc method header?

Reply:
Im not 100% on this but a javadoc doesnt feel like the best place to explain why intellij is linting a warning. I think an inline comment like this should be preferred.